style.css

```diff
@@ -28,10 +28,33 @@ header {
   border-bottom-left-radius: 7%;
   background-color: white;
   margin-bottom: 50px;
+  position: relative;
 }
 
 .nav-link {
-  display: none;
+  position: absolute;
+  width: 100%;
+  background-color: rgba(96, 112, 255, 0.9);
+  height: 100%;
+  left: 0;
+  top: 0;
+  padding-top: 100px;
+  padding-left: 28px;
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  transition: all 0.6s ease-in;
+}
+
+.hide-link {
+  height: 0;
+  overflow: hidden;
+  padding: 0;
+}
+
+.close-container {
+  position: absolute;
+  z-index: 100;
 }
```

Comment:
- [ ] Please make sure the mobile menu background has a blur effect and make sure it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=K0RVfXGn3ETgZ4Q2-0) design.
  - For blur effect, you can use `filter: blur(radius;`.

### Your design

![image](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/83803414/ddca8dea-e0ee-4960-a9e4-cfb615e0a687)

### Figma design

![image](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/83803414/e5922986-7a20-49f8-8761-cf7efb33d837)